website/docs/sql-reference/joins/sql-full-outer-join.md

```diff
@@ -0,0 +1,72 @@
+---
+id: outer-join
+title: SQL OUTER JOIN
+description: A SQL full outer join is a join between two tables that returns all rows from both tables, regardless of join key match success
+slug: /sql-reference/outer-join
+---
+
+<head>
+    <title>Working with full outer joins in SQL</title>
+</head>
+
+SQL full outer joins exist and therefore we have to talk about them, but they’re *highly unlikely* to be a join you regularly leverage in your data work. In plain terms, a SQL full outer join is a join between two tables that returns *all rows* from both tables, regardless of join key match success; compare this to [left](/sql-reference/left-join), [inner](/sql-reference/outer-join), or [right joins](/sql-reference/right-join) that require matches to be successful to return certain rows.
+
+In this page, we’ll unpack how to create a full outer join and demonstrate when you might need one in your analytics engineering work.
+
+## How to create a full outer join
+
+Like all joins, you need some database objects (ie tables/views), keys to join on, and a [select statement](/sql-reference/select) to perform a full outer join:
+
+```
+select
+    <fields>
+from <table_1> as t1
+full outer join <table_1> as t2
+on t1.id = t2.id 
+```
+
+In this example above, there’s only one field being used to join the table together; if you’re joining between database objects that require multiple fields, you can leverage AND/OR operators, and more preferably, <Term id="surrogate-key">surrogate keys</Term>. You may additionally add [WHERE](/sql-reference/where), [GROUP BY](/sql-reference/group-by), [ORDER BY](/sql-reference/order-by), [HAVING](/sql-reference/having), and other clauses after your joins to create filtering, ordering, and performing aggregations.
+
+A note on full outer joins: it may sound obvious, but because full outer joins can return all rows between two tables, they therefore can return *many* rows, which is not necessarily a recipe for efficiency. When you use full outer joins, you often can find alternatives using different joins or unions to potentially bypass major inefficiencies caused by a full outer join.
+
+### SQL full outer join example
+
+Table A `car_type`
```

Comment:
these tables are not being formatted properly 
<img width="686" alt="image" src="https://user-images.githubusercontent.com/18452766/213212140-423602d7-9978-46d1-a92e-d88809b6c6dd.png">